src/main/java/com/linkedin/kmf/apps/MultiClusterMonitor.java

```diff
@@ -29,17 +30,18 @@
 public class MultiClusterMonitor implements App {
   private static final Logger LOG = LoggerFactory.getLogger(MultiClusterMonitor.class);
 
-  private final MultiClusterTopicManagementService _topicManagementService;
+  private final MultiClusterTopicManagementService _multiClusterTopicManagementService;
   private final ProduceService _produceService;
   private final ConsumeService _consumeService;
   private final String _name;
 
   public MultiClusterMonitor(Map<String, Object> props, String name) throws Exception {
     _name = name;
     MultiClusterMonitorConfig config = new MultiClusterMonitorConfig(props);
-    _topicManagementService = new MultiClusterTopicManagementService(createMultiClusterTopicManagementServiceProps(props, config), name);
+    _multiClusterTopicManagementService = new MultiClusterTopicManagementService(createMultiClusterTopicManagementServiceProps(props, config), name);
+    CompletableFuture<Void> topicPartitionReady = _multiClusterTopicManagementService.topicPartitionReady();
```

Comment:
method name and future name `topicPartitionReady` is a little confusing. This operation could fail so consider naming the method `startBlahBlahAsync` and the future `blahBlahResult`.

Reply:
Stylistic concerns including method and variable naming conventions/lambda will be included in a separate PR.